app/views/ReportView/components/GenomicSummary/index.tsx

```diff
@@ -313,11 +318,11 @@ const GenomicSummary = ({
         },
         {
           term: 'Genome SNV TMB (mut/mb)', // float
-          value: tmburMutBur?.genomeIndelTmb.toString(),
+          value: tmburMutBur ? tmburMutBur.genomeIndelTmb.toString() : '',
```

Comment:
this change is that if there are no tmburMutBur data, this field would not even display